rmf_fleet_adapter/include/rmf_fleet_adapter/agv/Adapter.hpp

```diff
@@ -102,15 +103,76 @@ class Adapter : public std::enable_shared_from_this<Adapter>
   /// Create a traffic light to help manage robots that can only support pause
   /// and resume commands.
   ///
+  /// \param[in] command
+  ///   The handle for the traffic light to use to trigger commands. You must
+  ///   implement the TrafficLight::CommandHandle abstract interface for your
+  ///   robot or fleet.
+  ///
   /// \param[in] fleet_name
   ///   The name of the fleet
+  ///
+  /// \param[in] robot_name
+  ///   The name of the robot
+  ///
+  /// \param[in] traits
+  ///   The traits of the robot
+  ///
+  /// \param[in] handle_cb
+  ///   The callback that will be triggered when the traffic light handle is
+  ///   ready to be used
   void add_traffic_light(
       std::shared_ptr<TrafficLight::CommandHandle> command,
       const std::string& fleet_name,
       const std::string& robot_name,
       rmf_traffic::agv::VehicleTraits traits,
       std::function<void(TrafficLight::UpdateHandlePtr handle)> handle_cb);
 
+  using Blockers = std::vector<TrafficLight::CommandHandle::Blocker>;
+
+  /// Create an easy-to-use version of a traffic light to help manage robots
+  /// that can only support pause and resume commands.
+  ///
+  /// This API is much simpler to use than the standard traffic light API, but
+  /// it provides less information about the exact timing needed for the starts
+  /// and stops.
+  ///
+  /// This API should only be used for demo purposes, or if system integrators
+  /// can ensure very low-latency and reliable connections to the robots to
+  /// ensure that the commands arrive on time.
+  ///
```

Comment:
missing description for `\handle_callback` param

Reply:
Added!